geometry_msgs/Pose qs
geometry_msgs/Pose qg
nav_msgs/OccupancyGrid map
---
Trajectory trajectory
